generic_relations/tests/test_relations.py

```diff
@@ -212,10 +214,32 @@ class Meta:
         ]
         self.assertEqual(serializer.data, expected)
 
+    def test_deprecated_methods_overridden(self):
+        with warnings.catch_warnings(record=True) as w:
+            class MyRelatedField(GenericRelatedField):
+                def determine_deserializer_for_data(self, value):
+                    return super(MyRelatedField, self).determine_deserializer_for_data(value)
+                def determine_serializer_for_data(self, value):
+                    return super(MyRelatedField, self).determine_serializer_for_data(value)
+
+            self.assertEqual(len(w), 2)
```

Comment:
I actually made them `UserWarning`s since `DeprecationWarning` tends to pass silently (it's muted by default). (Django also does that.)

I'll add a type check here.